siad/main.go

```diff
@@ -157,6 +158,7 @@ func main() {
 	root.Flags().StringVarP(&globalConfig.Siad.RPCaddr, "rpc-addr", "", ":9981", "which port the gateway listens on")
 	root.Flags().StringVarP(&globalConfig.Siad.Modules, "modules", "M", "cghmrtw", "enabled modules, see 'siad modules' for more info")
 	root.Flags().BoolVarP(&globalConfig.Siad.AuthenticateAPI, "authenticate-api", "", false, "enable API password protection")
+	root.Flags().BoolVarP(&globalConfig.Siad.AllowAPIBind, "disable-security", "", false, "allow siad to listen on a non-localhost address (DANGEROUS)")
```

Comment:
given that this only applies to the api, I would mark it as "disable-api-security".

I was also thinking we could have a flag to enable the user to view the wallet seeds. Right now, if the wallet is unlocked you can fetch the wallet seeds over the API. This should be disabled by default, would prevent a lot of the repeated attacks that we're seeing I think.